pkg/securitypolicy/securitypolicy.go

```diff
@@ -361,7 +363,8 @@ func newOptionsFromConfig(mCfg *MountConfig) []string {
 // mount type.
 func newMountTypeFromConfig(mCfg *MountConfig) string {
 	if strings.HasPrefix(mCfg.HostPath, guestpath.SandboxMountPrefix) ||
-		strings.HasPrefix(mCfg.HostPath, guestpath.HugePagesMountPrefix) {
+		strings.HasPrefix(mCfg.HostPath, guestpath.HugePagesMountPrefix) ||
+		strings.HasPrefix(mCfg.HostPath, plan9Prefix) {
```

Comment:
stupid question: do we lowercase anywhere? or it assumed to be case-sensitive?

Reply:
no, I don't think we lowercase anywhere. we assume that the tooling handles that properly.